.gitignore

```diff
@@ -4,3 +4,4 @@ log
 yarn-error.log
 public/assets
 coverage
+.bundle
```

Comment:
Could you end the file with a newline?  I think most people's editors do that by default, so not doing so will cause some extra diff when someone next changes this.

Reply:
Oops!